codecov.yml

```
@@ -18,3 +40,24 @@ ignore:
   - web/.storybook
   - web/bin
   - web/src/gql/graphql-client-api.tsx
+
+flags:
+  web-classic:
+    paths:
+      - web/src/classic
+    carryforward: true
+  web-beta:
+    paths:
+      - web/src/beta
+      - web/src/services
+    carryforward: true
+    ignore:
+      - web/src/beta/utils
+  web-utils:
+    paths:
+      - web/src/beta/utils
```

```suggestion
      - web/src/beta/**/utils
```
I think we should check all utils. What do you think?